### ISSUE TITLE
[ui5-table compat]: table header row focus indicator outline is not fully visible

### ISSUE THREAD

**Issue by caroline-ferri** [SAP/ui5-webcomponents#11112]
### Bug Description

When the user tabs to a ui5-table from ui5-webcomponents-compat, the header focus outline is not fully visible. 

### Affected Component

ui5-table

### Expected Behaviour

Header focus outline should be fully visible. 

### Steps to Reproduce

1. Tab to table header row
2. view focus outline
3.
...

### Priority

None

### UI5 Web Components Version

2.7.2

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
